Cargo.toml

```diff
@@ -14,6 +14,8 @@ libc = "0.2"
 aligned_alloc = "0.1"
 num-complex = { version = "0.1.40", default-features = false }
 tensorflow-sys = { version = "0.12.0", path = "tensorflow-sys" }
+byteorder = "*"
+crc32c = "*"
```

Comment:
I honestly can't find a stable crc32c crate (everyone's using feature gates to get assembly instructions). I'll probably do one myself.

Reply:
What about the `crc` crate?  It has a thousand times as many downloads as `crc32c` and doesn't seem to require anything but the stable channel.  I don't mind requiring a different release channel if it's feature-gated, but I think we've been doing well to stick to the stable channel for default builds.

Reply:
I'll take a look.